assets/components/src/accordion/index.js

```diff
@@ -24,7 +24,7 @@ const Accordion = ( { children, title } ) => {
 				{ title }
 				<Icon className="newspack-accordion__icon" icon={ chevronRight } size={ 24 } />
 			</summary>
-			{ children }
+			<div className="newspack-accordion__content">{ children }</div>
```

Comment:
Note: for this change and the one in `assets/components/src/accordion/style.scss`, we didn't end up needing to us the Accordion component in this PR, but I'm retaining the changes because it does fix some display issues in the (currently unused) component.